src/Console/Command/FixCommand.php

```diff
@@ -157,6 +157,8 @@ public function getHelp(): string
 
 * <comment>none</comment>: disables progress output;
 * <comment>dots</comment>: multiline progress output with number of files and percentage on each line.
+* <comment>detail</comment>: multiline progress output with status, path and applied fixers for each file.
+* <comment>meaningful</comment>: multiline progress output with status, path and applied fixers for each file with meaningful result (non-modified or skipped are ignored).
```

Comment:
should we have some kind of progress indicator? otherwise, assuming no-issues in big codebase case, i will trigger command, wait minute, see it finished - yet, no indicator how far it is processing things or how much is still there to be done

Reply:
Exactly the same is now without explicit `--show-progress=dots`, so I think it's enough. Users who choose this progress output know what is expected 🙂.

Reply:
I was thinking about adding one-line progress bar to stderr (eg https://symfony.com/doc/current/components/console/helpers/progressbar.html ) and....
decide if we want to switch the default behaviour to this one

Reply:
In terms of `detail` and `meaningful` outputs I decided to not include progress there because for the former it only would make sense to make it on the left (on the right would be much more complicated: determining console and text width, calculating whitespace required to fill space between text and progress, verbose output with applied fixers would require also handling multiline output for that), and for the latter it wouldn't make much sense because output is filtered to meaningful results only so progress would jump between random numbers. I think I tried to use sections and print progress bar AND the current output, but there was some technical issue with it.

I can try again, but I would prefer doing it in separate PR because it shouldn't block outputs proposed here 🙂. Let's focus on what's here, if it's sufficient or has to be improved.

Reply:
@keradus progress bar was introduced in #7603 🙂. Let's merge that one since it's straightforward, and I'll rebase this one to keep it up-to-date.